cypress.env.json

```diff
@@ -5,5 +5,6 @@
   "Button": "http://localhost:9001/?path=/story/button--default",
   "Button Toggle":"http://localhost:9001/?path=/story/button-toggle--default",
   "Button Toggle Group": "http://localhost:9001/?path=/story/button-toggle-group--default",
-  "Dialog Full Screen": "http://localhost:9001/?path=/story/dialog-full-screen--default"
+  "Dialog Full Screen": "http://localhost:9001/?path=/story/dialog-full-screen--default",
+  "Dialog": "http://localhost:9001/?path=/story/dialog--default"
```

Comment:
This file has a lot of repetition, and will keep growing as we add test suites. 

I would reorganise this so that there is only one `root` path: `expect(instance.formatValue()).toEqual('1,234,567.00')`, and have a function add the name of the story to the end.

Reply:
fixed